src/examples/src/widgets/button/DisabledSubmit.tsx

```diff
@@ -0,0 +1,12 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import Button from '@dojo/widgets/button';
+
+const factory = create();
+
+export default factory(function DisabledSubmitButton() {
+	return (
+		<Button type="submit" disabled={true}>
```

Comment:
this can just be `disabled`, no need for the `={true}`

Reply:
fixed